server/models/Database.js

```diff
@@ -0,0 +1,9 @@
+class Database {
```

Comment:
For now no need to add such complexity with database(class with inheritance), please remove models/Database.js, models/FirebaseDatabase.js and create databaseConfig(outside of models folder as technically it's not model😉)